kf_utils/dataservice/delete.py

```diff
@@ -119,30 +83,20 @@ def delete_entities(host, study_ids=None, safety_check=True):
     :type study_ids: list of str
     :param saftey_check: Whether to delete if resource is not at localhost
     :type safety_check: bool
-
-    :returns: Dict of errors where Keys are urls that failed delete (non 200
-    status code) and values are requests.Response objects
     """
     phrase = f"studies {pformat(study_ids)}" if study_ids else "all studies"
-    logger.info(f"Deleting {phrase} from {host}")
+    print(f"Deleting {phrase} from {host}")
 
     # Get all study ids
     if not study_ids:
         study_ids = yield_kfids(host, "studies", {})
 
     # Delete entities by study id
-    errors = {}
     for study_id in study_ids:
         # Delete entities except "study" (it has to be handled differently)
         for endpoint in ENDPOINTS:
-            params = {"study_id": study_id}
-            kfids = yield_kfids(host, endpoint, params)
-            errors.update(
-                delete_kfids(host, endpoint, kfids, safety_check=safety_check)
-            )
-        # Delete study by its kfid
-        errors.update(
-            delete_kfids(host, STUDIES, [study_id], safety_check=safety_check)
-        )
+            kfids = yield_kfids(host, endpoint, {"study_id": study_id})
+            delete_kfids(host, kfids, safety_check=safety_check)
```

Comment:
`delete_kfids` prints how many things are being deleted at the current endpoint. there should be a statement before deleting printing what endpoint is currently being deleted e.g:
```suggestion
            print(f"Deleting kfids from {endpoint}.")
            delete_kfids(host, kfids, safety_check=safety_check)
```

Otherwise it's not clear where in the process of deleting the script is.